terraformutils/utils.go

```diff
@@ -96,6 +96,8 @@ func RefreshResources(resources []*Resource, provider *providerwrapper.ProviderW
 
 	wg.Wait()
 	for _, r := range resources {
+		log.Printf("Instance state %v  \n", r.InstanceState)
```

Comment:
please delete, no need here

Reply:
@sergeylanzman done in c07585a114788b9a428a7d601d501196d4306e73